sdk/core/azure-core/src/main/java/com/azure/core/http/policy/RetryStrategy.java

```diff
@@ -57,6 +60,12 @@ default boolean shouldRetry(HttpResponse httpResponse) {
      * @return Whether a retry should be attempted.
      */
     default boolean shouldRetryException(Throwable throwable) {
-        return throwable instanceof Exception;
+        if ((throwable instanceof HttpResponseException)) {
+            int statusCode = (((HttpResponseException) throwable).getResponse().getStatusCode());
+
+            return statusCode != 401 && statusCode != 403;
```

Comment:
This should get passed into the `shouldRetry(HttpResponse)` method instead of duplicating the logic